### PR TITLE
Dog treats

### PR DESCRIPTION
**Pull request by FlattestGuitar**
:cl: Flatty
add: As per HR guidelines, Nanotrasen has introduced a dog treat recipe to the standard repertoire of the station canteen. Bon Appétit!
/:cl:

Good boys are a severely underrepresented demographic on oracle. This needs to be amended.

Humans puke from these, Unathi and Ethari love 'em.

### CODE REVIEW

**Comment by AffectedArc07** [comment]
I am pissed for multiple reasons
- Dog treats
- Ethari are being removed
- YOU WROTE BON APPETIT NOT BONE APPETIT YOU MISSED OUT ON SUCH A GOOD JOKE

**Comment by swagbringer** [comment]
These are the changes Ethari mains have been waiting for.

**Comment by AndrewMontagne** [comment]
April fools was four days ago, @FlattestGuitar

**Comment by Cactus2388** [comment]
s t o p

**Comment by Ralta** [comment]
https://youtu.be/sO1QIUj0M4k

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm [hunk 1]
```diff
@@ -170,3 +170,11 @@
...
 	result = /obj/item/reagent_containers/food/snacks/pigblanket
 	subcategory = CAT_MEAT
+
+/datum/crafting_recipe/food/dogtreat
+	name = "Dog Treats"
+	reqs = list(
+		/obj/item/reagent_containers/food/snacks/meat/slab = 1
+		)
+	result = /obj/item/reagent_containers/food/snacks/dogtreat
+	subcategory = CAT_MEAT
```
Comment (Lorwp) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm]: >Dog treats are actually rebranded cat food

oh darn

File: code/modules/food_and_drinks/food/snacks_pastry.dm [hunk 2]
```diff
@@ -219,6 +219,16 @@
 	tastes = list("pretzel" = 1)
 	foodtype = GRAIN | SUGAR
 
+/obj/item/reagent_containers/food/snacks/dogtreat
+	name = "dog treat"
+	desc = "For good boys only."
```
Comment (taukausanake) [inline code/modules/food_and_drinks/food/snacks_pastry.dm]: Pretty sure a majority of fur characters are female, so this is sexist as fuuuuuck.

But I could be wrong

**Comment by Ziiro** [comment]
no

**Comment by scrubmcnoob** [comment]
so when is protein powder getting added in

File: code/modules/food_and_drinks/food/snacks_pastry.dm [hunk 3]
```diff
@@ -219,6 +219,16 @@
 	tastes = list("pretzel" = 1)
 	foodtype = GRAIN | SUGAR
 
+/obj/item/reagent_containers/food/snacks/dogtreat
```
Comment (AffectedArc07) [inline code/modules/food_and_drinks/food/snacks_pastry.dm]: Remove

**Comment by SamHPurp** [comment]
Bad Flattest. No. Sit.

**Pull request closed without merging**